### ISSUE TITLE
Tmux status bar doesn't look as expected on macOS Sonoma 14.0

### ISSUE THREAD

**Issue by reckerp** [rose-pine/tmux#20]
# Issue

## Problem

On macOS Sonoma (14.0), the Tmux status bar doesn't display correctly when using the 'rose-pine/tmux' theme installed via Tmux Plugin Manager (tpm) and setting the 'main' variant.

## Current Behavior

The status bar is not appearing as expected, and the issue can be observed in the following screenshot:

![Tmux Issue Screenshot](https://github.com/rose-pine/tmux/assets/67198110/747acb54-4b10-4c7b-aaeb-c9e5042bced8)

## Steps to Reproduce

1. Add the following configuration to your Tmux setup to set the 'main' variant of the 'rose-pine' theme:

    ```tmux
    # Rose Pine theme
    set -g @rose_pine_variant 'main' # Options are 'main', 'moon' or 'dawn'

    # List of plugins
    set -g @plugin 'tmux-plugins/tpm'
    set -g @plugin 'tmux-plugins/tmux-sensible'
    set -g @plugin 'rose-pine/tmux'
    ```

2. Ensure that the 'rose-pine' theme is properly installed and the configuration file is correctly sourced.

3. Observe the appearance of the status bar.

## Expected Behavior

The Tmux status bar should display correctly with the 'main' variant of the 'rose-pine' theme, as it is shown in the screenshots in the readme.

## Additional Information

- The issue was initially encountered while experimenting with rose-pine tmux settings and resulted in the screenshot.
- Simplifying the configuration to only include setting the 'main' variant produced the same unexpected result.
- The configuration works as expected on my Debian server, suggesting that the issue may be specific to macOS Sonoma (14.0).
- I utilize iTerm2 on my macOS, and I have configured it with the 'rose-pine' theme from the same GitHub user. The issue is replicated when using the default Apple Terminal application.
- Furthermore, I have a Nerd Font installed, specifically Jetbrains Mono NF.
- It's worth noting that when using iTerm2 and connecting to the Debian system via ssh, the configuration works as expected.

**Comment by mrs4ndman** [comment]
Don't have a macOS system at hand so let's take some things out of the equation:
- What version of tmux are you running on macOS? I have tested on v3.2a, so I might be missing some compatibility issues
- A minimal tmux config with just the theme would look like this. Try it and let me know:
```tmux
# List of plugins
set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'tmux-plugins/tmux-sensible'
set -g @plugin 'rose-pine/tmux'

# Rose Pine theme
set -g @rose_pine_variant 'main' # Options are 'main', 'moon' or 'dawn'


run '~/.tmux/plugins/tpm/tpm'
```

**Comment by reckerp** [comment]
Hi,
- currently I run version 3.3a on Mac and 3.1c on Debian
- I tried exactly running your minimal setup with the same result.

**Comment by reckerp** [comment]
Update:
- After uninstalling tmux, deleting tpm, deleting the tmux/tmux.conf file, restarting my Mac and installing + configuring everything again the basic config works now. 
- It was really weird caching issue, I guess. I tried uninstalling twice, once without restarting my Mac and the second time with a restart, which fixed the problem
<img width="581" alt="Bildschirmfoto 2023-10-11 um 22 04 52" src="https://github.com/rose-pine/tmux/assets/67198110/77f0189f-2883-4c24-ba19-c19b11f12338">


- Now I found out what's the cause for the problem `set -g @rose_pine_only_windows 'on' # Leaves only the window module, for max focus and space
` 
- Having this in my config causes the error in issue in the first screenshot. 

- The other config works for me know. Apparently only a caching error
